app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -3,7 +3,7 @@
 // docs/CHANGELOG-JAVASCRIPT-PLUGIN-API.md whenever you change the version
 // using the format described at https://keepachangelog.com/en/1.0.0/.
 
-export const PLUGIN_API_VERSION = "1.38.0";
+export const PLUGIN_API_VERSION = "1.37.4";
```

Comment:
Looks like core got bumped, so we'll need to bump it higher in this branch

Reply:
Done